css-two-column-layout/styles.css

```diff
@@ -0,0 +1,82 @@
+p,
+h3,
+h1 {
+  font-family: Karla, sans-serif;
+}
+
+h1 {
+  text-transform: uppercase;
+  font-size: 63px;
+  margin-top: 100px;
+  margin-bottom: 22px;
+  letter-spacing: 0.25rem;
+}
+
+p {
+  line-height: 1.75em;
+  font-size: 1.15em;
+  color: #5f5f5f;
+}
+
+.header-column-full > p {
+  font-size: 1.2em;
+}
+
+h3 {
+  text-transform: uppercase;
+  letter-spacing: 0.1rem;
+}
+
+.header-column-full {
+  text-align: center;
+}
+
+.container {
+  margin: 0 auto;
+  width: 100%;
+}
+
+.row-2 {
+  display: flex;
+}
```

Comment:
What happened to `.row`? You should never make numbered classes, it isn't descriptive of their purpose at all.